schema/cassandra/cadence/versioned/v0.27/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CurrVersion": "0.27",
```

Comment:
Why 0.27? I think we only increase version once per release. 0.12.x is using 0.25, so I guess we can keeping 0.26 here. You can have multiple cql file in one version.

Reply:
I am thinking about having the 0.26 in 0.12 release. Let's discuss offline.

Reply:
I see. Make sense.